ava-e2e.config.js

```diff
@@ -0,0 +1,6 @@
+// Calculate nodeArguments based on the Node version
```

Comment:
This comment is obsolete, also in the original file `ava.config.js`

Reply:
Done